examples/Cargo.toml

```diff
@@ -7,5 +7,5 @@ edition = "2018"
 [dependencies]
 winit = "0.25.0"
 image = "0.10.4"
-ash = { path = "../ash" }
+ash = { path = "../ash", default-features = false, features = ["linked", "debug"] }
```

Comment:
It might be helpful to have some explanation as to why the examples are using `linked` instead `loaded`. I assume this is because the examples are requiring the validation layer from the SDK. If that's not the case, the examples should probably use `loaded` to minimize friction.

Reply:
Added a note.

Reply:
@aloucks The note is nice, but keep in mind that dynamic linking or (dynamic) runtime loading has no impact on being able to use the validation layers. It "just so happens" that both the `.lib` (needed for dynamic linking on Windows) (on Linux the runtime `.so` is used for this) and validation layers ship with the SDK.

FWIW I thought the examples would use `linked` to both showcase this feature, and because examples are generally ran on the same machine (ie. build environment == runtime environment) so it makes sense that the user intends to develop Vulkan applications, hence has the ability to both "develop" (with the SDK) and run them.